spicy/toolchain/src/compiler/codegen/parser-builder.cc

```diff
@@ -337,7 +337,7 @@ struct ProductionVisitor
                             builder()->addLocal("filtered_data", type::ValueReference(type::Stream()),
                                                 builder::id("filtered"));
                             args2[0] = builder::id("filtered_data");
-                            args2[1] = builder::optional(type::stream::Iterator());
+                            args2[1] = builder::default_(type::stream::Iterator()); // FIXME(bbannier): sensible data.
```

Comment:
What does this mean?

Reply:
Thanks for catching this! This was leftover WIP code for getting the typing right which suprisingly seems to have had no effect with our test suite. I added the correct value which we already have available.